packages/nc-gui-v2/composables/useProject.ts

```diff
@@ -11,6 +11,7 @@ export function useProject(projectId?: MaybeRef<string>) {
   const _projectId = $computed(() => unref(projectId))
 
   const project = useState<ProjectType>('project')
+  const appInfo = useState<any>('appInfo')
```

Comment:
Is this still necessary or can this be removed?
`appInfo` seems to be stored in the global state now so this seems like two states.